extendr-engine/README.md

```diff
@@ -1,4 +1,26 @@
-# extendr-engine
+# `extendr-engine`
 
-This crate is mostly use for testing `extendr-api`. See
-`extendr-api` for more details.
+This crate facilitates embedding an R process together with a Rust binary.
+It is not meant to be used within rust-powered R-packages. Rather, this crate
```

Comment:
This sentence doesn't seem to jive with the next section.

Reply:
Resolving this, because I've pushed a change. But I think you're meant to raise the issue again after my change, in a new thread, if the issue persist.